fastlane/Fastfile

```diff
@@ -0,0 +1,74 @@
+default_platform(:mac)
+
+platform :mac do
+	GITHUB_REMOTE = "origin"
+	RELEASE_BRANCH = "master"
+	GITHUB_REPOSITORY = "aaronraimist/seaglass"
+
+	VERSION = sh "bash ../scripts/version.sh"
+	ZIP_PATH = "/tmp/seaglass/upload/Seaglass-#{VERSION}.zip"
+
+	desc "Build and release Seaglass, this is the lane you probably want to use"
+	lane :build_and_release do
+		build
+
+		#ensure_git_branch(branch: "master")
```

Comment:
Turns out this results in a hard fail of the CI task so modified `.circleci/config.yml` instead to check the branch before running `build_and_release`.